pulsar/managers/base/__init__.py

```diff
@@ -38,7 +38,7 @@
 ID_ASSIGNER = {
     # Generate a random id, needed if multiple
     # Galaxy instances submitting to same Pulsar.
-    'uuid': lambda galaxy_job_id: uuid4().hex,
+    'uuid': lambda galaxy_job_id: "{}-{}".format(galaxy_job_id, uuid4().hex),
```

Comment:
Can you create a new type - ``job_plus_uuid`` for this? I feel like ``uuid`` should stick with uuids as job IDs and maybe just document what other parameters it doesn't work with for now?